packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -58,8 +57,11 @@ export const sendResponseEmail = async (
   ownerEmail: string
 ) => {
   try {
-    const email = new ResponseEmail({ form: form, toAddresses: [ownerEmail], response: response });
-    await email.sendEmail();
+    if (typeof window === "undefined") {
+      const ResponseEmail = (await import("../emails/templates/response-email")).default;
```

Comment:
Even though this function isn't included browser side, I don't know why it tries to be built for client side code and thus throwing the error.

So, I have to do it.